rebasehelper/options.py

```diff
@@ -181,13 +181,6 @@
         "dest": "disable_inapplicable_patches",
         "help": "disable inapplicable patches in rebased SPEC file",
     },
-    {
-        "name": ["--keep-comments"],
```

Comment:
Specfile keeps the comments by default right? I am wondering if there shouldn't be an option to override this behaviour. On the other hand, keeping the comments is usually desired and will result in a "false positive" not that often, I assume

Reply:
No, vice versa, specfile removes associated comments when removing a source/patch. And it doesn't make much sense to me that someone would want the comments to be left in the spec file (since they are tied to the source/patch being removed).

Reply:
I see you added the option in https://github.com/rebase-helper/rebase-helper/commit/4eef5e263ae8892dbbf2d82c07a2e507ab6fb282. Do you remember what for?

Reply:
Oh, yes, I got a bit confused. Right, it doesn't make sense to keep the comments. The commit was trying to solve https://github.com/rebase-helper/rebase-helper/issues/826 but I am not sure why I added the option to keep the comments. Maybe for backwards compatibility so that the users could change the behavior to the old one if the removal made some mess (e.g. the comment looked to be related to the patch but semantically, it wasn't). I am fine with dropping the option.